conbench/hacks.py

```diff
@@ -46,6 +46,15 @@ def set_display_batch(benchmark):
         benchmark.display_batch = batch
 
 
+def set_tags(benchmark):
```

Comment:
This isn't needed.

If you apply this diff, I think you'll find all tests pass.

```
diff --git a/conbench/api/compare.py b/conbench/api/compare.py
index 9fd4bbf..b5775e8 100644
--- a/conbench/api/compare.py
+++ b/conbench/api/compare.py
@@ -6,7 +6,7 @@ from ..api._comparator import BenchmarkComparator, BenchmarkListComparator
 from ..api._endpoint import ApiEndpoint
 from ..entities._entity import NotFound
 from ..entities.summary import Summary
-from ..hacks import set_display_batch, set_display_name, set_tags
+from ..hacks import set_display_batch, set_display_name
 
 
 def _compare_entity(summary):
@@ -18,7 +18,7 @@ def _compare_entity(summary):
         "unit": summary.unit,
         "benchmark": summary.display_name,
         "batch": summary.display_batch,
-        "tags": summary.tags,
+        "tags": summary.case.tags,
     }
 
 
@@ -71,8 +71,6 @@ class CompareBenchmarksAPI(ApiEndpoint):
         set_display_name(contender_summary)
         set_display_batch(baseline_summary)
         set_display_batch(contender_summary)
-        set_tags(baseline_summary)
-        set_tags(contender_summary)
 
         baseline = _compare_entity(baseline_summary)
         contender = _compare_entity(contender_summary)
@@ -143,12 +141,10 @@ class CompareBatchesAPI(ApiEndpoint):
         for summary in baselines:
             set_display_name(summary)
             set_display_batch(summary)
-            set_tags(summary)
             self._add_pair(pairs, summary, "baseline")
         for summary in contenders:
             set_display_name(summary)
             set_display_batch(summary)
-            set_tags(summary)
             self._add_pair(pairs, summary, "contender")
 
         if raw:
diff --git a/conbench/hacks.py b/conbench/hacks.py
index e536d1e..7b72b99 100644
--- a/conbench/hacks.py
+++ b/conbench/hacks.py
@@ -46,15 +46,6 @@ def set_display_batch(benchmark):
         benchmark.display_batch = batch
 
 
-def set_tags(benchmark):
-    is_api = isinstance(benchmark, dict)
-    tags = benchmark["tags"] if is_api else benchmark.case.tags
-    if is_api:
-        benchmark["tags"] = tags
-    else:
-        benchmark.tags = tags
-
-
 def sorted_data(benchmarks):
     data = []
     for benchmark in benchmarks:
```